vagrant_support/repl.rc

```diff
@@ -0,0 +1,28 @@
+#!perl
+# vim: set ft=perl:
+use Bugzilla;
+use Bugzilla::Constants;
+Bugzilla->usage_mode(USAGE_MODE_CMDLINE);
+Bugzilla->extensions;
+use DDP return_value => 'void', colored => 1;
```

Comment:
Conider giving this an extra newline between +6 and +7, so that the DDP segment is clearly distinct from the Bugzilla segment.